src/base/dof_map.C

```diff
@@ -1080,6 +1080,9 @@ void DofMap::distribute_dofs (MeshBase & mesh)
   // EquationSystems call that for us, after we've added constraint
   // dependencies to the send_list too.
   // this->sort_send_list ();
+
+  // Return total number of DOFs across all procs.
+  return std::accumulate(dofs_on_proc.begin(), dofs_on_proc.end(), static_cast<std::size_t>(0));
```

Comment:
Might want a comment here about the distinction between the dof_id_type calculation we did earlier to get _end_df[n_proc-1] and the forced-to-be-size_t calculation here; neither this code nor its motivation would be obvious to me if we hadn't just been talking about it.